src/module/system/text-editor.ts

```diff
@@ -676,7 +682,8 @@ async function augmentInlineDamageRoll(
             "inline-damage",
             item ? `${item.id}-inline-damage` : null,
             item ? `${sluggify(item.slug ?? item.name)}-inline-damage` : null,
-        ]);
+            args.domains,
```

Comment:
Can this be vetted?

Reply:
Its vetted in the params, but we could move the point of validation to augmentDamageRoll() if you prefer.